docs/UserGuide.md

```diff
@@ -205,7 +206,7 @@ _Details coming soon ..._
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]… [c/COMMENT]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague c/Owes a cookie`
+**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/telegramHandle [t/TAG]… [c/COMMENT]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/jamesTele t/friend t/colleague c/Owes a cookie`
```

Comment:
Would it be better if you changed this to a/TELEGRAM_HANDLE to stay consistent with all the other fields?